File: python_programming_basics/conditional_statements_advanced/e_03_new_house.py
flower_type = input()
flowers_count = int(input())
budget = int(input())
result = 0

if flower_type == "Roses":
    result = flowers_count * 5

    if flowers_count > 80:
        result -= result * 0.1

elif flower_type == "Dahlias":
    result = flowers_count * 3.8

    if flowers_count > 90:
        result -= result * 0.15

elif flower_type == "Tulips":
    result = flowers_count * 2.8

    if flowers_count > 80:
        result -= result * 0.15

elif flower_type == "Narcissus":
    result = flowers_count * 3

    if flowers_count < 120:
        result += result * 0.15

elif flower_type == "Gladiolus":
    result = flowers_count * 2.5

    if flowers_count < 80:
        result += result * 0.2

if budget >= result:
    print(f"Hey, you have a great garden with {flowers_count} {flower_type} and {budget - result:.2f} leva left.")

else:
    print(f"Not enough money, you need {result - budget:.2f} leva more.")

File: JavaScript Basics/03. Conditional Statements Advanced/e-03-New-House.js
function calculateFlowerCost(flowerType, flowerCount, budget) {
    let flowerPrice = 0;
    let totalPrice = 0;

    if (flowerType === "Roses") {
        flowerPrice = 5;
    } else if (flowerType === "Dahlias") {
        flowerPrice = 3.80;
    } else if (flowerType === "Tulips") {
        flowerPrice = 2.80;
    } else if (flowerType === "Narcissus") {
        flowerPrice = 3;
    } else if (flowerType === "Gladiolus") {
        flowerPrice = 2.50;
    }

    totalPrice = flowerPrice * flowerCount;

    if (flowerType === "Roses" && flowerCount > 80) {
        totalPrice *= 0.90;
    } else if (flowerType === "Dahlias" && flowerCount > 90) {
        totalPrice *= 0.85;
    } else if (flowerType === "Tulips" && flowerCount > 80) {
        totalPrice *= 0.85;
    } else if (flowerType === "Narcissus" && flowerCount < 120) {
        totalPrice *= 1.15;
    } else if (flowerType === "Gladiolus" && flowerCount < 80) {
        totalPrice *= 1.20;
    }

   
    if (budget >= totalPrice) {
        let remainingMoney = (budget - totalPrice).toFixed(2);
        console.log(`Hey, you have a great garden with ${flowerCount} ${flowerType} and ${remainingMoney} leva left.`);
    } else {
        let neededMoney = (totalPrice - budget).toFixed(2);
        console.log(`Not enough money, you need ${neededMoney} leva more.`);
    }
}


calculateFlowerCost("Roses", 55, 250);
calculateFlowerCost("Tulips", 88, 260);
calculateFlowerCost("Narcissus", 119, 360);
